custom/icds_reports/sqldata/exports/growth_tracker_report.py

```diff
@@ -67,83 +61,38 @@ def _fetch_data(filters, order_by, case_by_grouping=False):
-                _check_case_presence(row['case_id'], 'pse_days_attended', data_month_1),
-                _check_case_presence(row['case_id'], 'pse_days_attended', data_month_2),
-                row['pse_days_attended'],
-                _check_case_presence(row['case_id'], 'lunch_count', data_month_1),
-                _check_case_presence(row['case_id'], 'lunch_count', data_month_2),
-                row['lunch_count'],
-                get_status(
-                    _check_case_presence(row['case_id'], current_month_stunting_column(self.beta),
-                                         data_month_1),
-                    'stunted',
-                    'Normal height for age',
-                    True
-                ),
-                get_status(
-                    _check_case_presence(row['case_id'], current_month_stunting_column(self.beta),
-                                         data_month_2),
-                    'stunted',
-                    'Normal height for age',
-                    True
-                ),
-                get_status(
-                    row[current_month_stunting_column(self.beta)],
-                    'stunted',
-                    'Normal height for age',
-                    True
-                ),
-                get_status(
-                    _check_case_presence(row['case_id'], current_month_wasting_column(self.beta),
-                                         data_month_1),
-                    'wasted',
-                    'Normal weight for height',
-                    True
-                ),
-                get_status(
-                    _check_case_presence(row['case_id'], current_month_wasting_column(self.beta), data_month_2),
-                    'wasted',
-                    'Normal weight for height',
-                    True
-                ),
-                get_status(
-                    row[current_month_wasting_column(self.beta)],
-                    'wasted',
-                    'Normal weight for height',
-                    True
-                ),
-                get_status(
-                    _check_case_presence(row['case_id'], 'current_month_nutrition_status', data_month_1),
-                    'underweight',
-                    'Normal weight for age',
-                    True
-                ),
-                get_status(
-                    _check_case_presence(row['case_id'], 'current_month_nutrition_status', data_month_2),
-                    'underweight',
-                    'Normal weight for age',
-                    True
-                ),
-                get_status(
-                    row['current_month_nutrition_status'],
-                    'underweight',
-                    'Normal weight for age',
-                    True
-                )
-            ]
-            excel_rows.append(row_data)
+            final_dict[case_id][k] = row['pse_days_attended']
+            final_dict[case_id][k + 3] = row['lunch_count']
```

Comment:
why its +3 and +6 here?

Reply:
for month_1 -> k = 9 -> cols -> 9,12,15,18,21 are updated 
      month_2 -> k = 10 -> cols -> 10,13,16,19,22 are updated
      month_3 -> k = 11 -> cols -> 11,14,17,20,23 are updated